src/clr-angular/utils/drag-and-drop/providers/drag-and-drop-event-bus.spec.ts

```diff
@@ -3,52 +3,59 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-
 import {ClrDragEvent, ClrDragEventType} from "../interfaces/drag-event";
 import {ClrDragAndDropEventBus} from "./drag-and-drop-event-bus";
 
 export default function(): void {
     describe("Drag And Drop Event Bus Provider", function() {
         const dndEventBus = new ClrDragAndDropEventBus();
+        const dragEventMockObj = (dragEventType: ClrDragEventType, dragDataTransfer?: any): ClrDragEvent<null> => {
```

Comment:
Shouldn't the type for `dragDataTransfer` match that of `ClrDragEvent`?

For eg: this is valid:
`const dragEventMockObj = (dragEventType: ClrDragEventType, dragDataTransfer?: string): ClrDragEvent<string>`

but you have `any` and `null` which do not match.